src/cdk/testing/test-harnesses.md

```diff
@@ -154,21 +177,22 @@ therefore, the Angular team recommends using
 to improve the test readability.
 
 Note that `await` statements block the execution of your test until the associated `Promise`
-resolves. When reading multiple properties of a harness it may not be necessary to block on the
-first before asking for the next, in these cases use `Promise.all` to parallelize.
-
-For example, consider the following example of reading both the `checked` and `indeterminate` state
-off of a checkbox harness:
+resolves. There are often times when you want to perform multiple actions simultaneously and wait
+until they're all done rather than blocking. For example, reading multiple properties off a harness.
+In these situations use the `parallel` function to parallelize the operations. The parallel function
+works similarly to `Promise.all`, while also optimizing change detection, so it is not run an
+excessive number of times. For example:
```

Comment:
```suggestion
resolves. Occasionally, you may want to perform multiple actions simultaneously and wait
until they're all done rather than performing each action sequentially. For example, reading multiple properties off a single components.
In these situations use the `parallel` function to parallelize the operations. The parallel function
works similarly to `Promise.all`, while also optimizing change detection, so it is not run an
excessive number of times. The following code demonstrates how you can read multiple properties from a harness with `parallel`:
```